### DISCUSSION TITLE
Unable to integrate trading view chart

### DISCUSSION THREAD

**Discussion by mohideenqiraad** [tradingview/lightweight-charts#1160] [Q&A]
- Currently, we are only able to generate the tradingview chart using “datafeeds” physical files and “charting_library”. But in our case we want to generate the tradingview chart using our own content and real data.
- Currently, we are using a free version of tradingview chart for our internal development and POC purposes. 
- So we are using “/charting_library/” for rendering the charts and this charting library is standalone.
- In order to make the charts more dynamic and real data, please provide the detailed integration steps in our ReactJS-based Fintech application.

**Comment by edew** [comment]
@mohideenqiraad 

Are you using Lightweight Charts? It seems you are talking about the charting library https://www.tradingview.com/HTML5-stock-forex-bitcoin-charting-library/?feature=technical-analysis-charts

**Reply by mohideenqiraad** [reply]
Yes we are using Lightweight Charts, but we are looking for Technical Analysis Charts. 
could you please share the details about implemented "Technical Analysis Charts" version?

**Reply by edew** [reply]
If you don't have access to the TAC then you can get access by following the instructions on this page https://www.tradingview.com/HTML5-stock-forex-bitcoin-charting-library/?feature=technical-analysis-charts
